### DISCUSSION TITLE
HTTP/3

### DISCUSSION THREAD

**Discussion by DregondRahl** [vercel/community#324] [Idea]
Hey, just wondering if Vercel will enable HTTP v3 for requests. Modern browsers can take advantage of the performance.

**Comment by colinmcdermott** [comment]
Vercel need to get this enabled, lagging behind the competition here.

**Comment by agam-riyandana** [comment]
Let's just hope that Vercel has HTTP V3 enabled

**Reply by colinmcdermott** [reply]
Vercel does not have HTTP/3 enabled https://twitter.com/ColinMcDermott/status/1687827911569444865
